lib/http_aws_sigv4.c

```diff
@@ -612,14 +609,14 @@ CURLcode Curl_output_aws_sigv4(struct Curl_easy *data, bool proxy)
                                "Credential=%s/%s, "
                                "SignedHeaders=%s, "
                                "Signature=%s\r\n"
-                               "%s\r\n"
+                               "%s" /* if providedd by user, must not be add */
```

Comment:
This comment does not make sense (apart from the broken English). If this header is provided by the user, it is not added *here* no, but it is still added to the request.

Reply:
I've just changed it to:
```C
                               /*                                                                                                                                                                           
                                * date_header is added here, only if it wasn't                                                                                                                              
                                * pass as header (using -H),                                                                                                                                           
                                * otherwise, date_header include \r\n                                                                                                                                       
                                */
```